src/plugins/flexsearch.js

```diff
@@ -8,18 +8,39 @@ import remarkParse from 'remark-parse';
 import { visit } from 'unist-util-visit';
 import { toString } from 'mdast-util-to-string';
 import { findAfter } from 'unist-util-find-after';
+import { findBefore } from 'unist-util-find-before';
 
 function slugify(str) {
   return str.toLowerCase().replace(/\s/g, '-');
 }
 
+function stripBadge(str) {
+  return str.replace(/<Badge.*>\s/, '');
+}
+
+function formatTitle(node, root) {
+  const nodeString = toString(node);
+
+  if (nodeString == 'Attributes') {
+    const parentHeading = stripBadge(toString(findBefore(root, node, 'heading')));
+    return stripBadge(`${parentHeading} ${nodeString}`);
+  }
+
+  if (nodeString == 'Errors') {
+    const headingNode = toString(findBefore(root, findBefore(root, node, 'heading'), 'heading'));
+    return stripBadge(`${headingNode} ${nodeString}`);
+  }
+
+  return stripBadge(nodeString);
```

Comment:
@alexBinns I can still see the badge, here is an example:
<img width="776" alt="image" src="https://github.com/centrapay/centrapay-docs/assets/64108933/ebebd5c2-8dcc-49f1-9faf-c32d3b590a92">

Reply:
There was an issue with my regex I have fixed now
![image](https://github.com/centrapay/centrapay-docs/assets/85718272/7eec1092-5384-4d32-9c00-c5153f656b9d)